src/components/DayTemperature/index.js

```diff
@@ -0,0 +1,29 @@
+import "./DayTemperatureBlock.scss";
+import dayjs from 'dayjs'
+import { getTemperature } from "../../utilities/helpers/get-temperature";
+
+export const DayTemperatureBlock = ({ daysTemperature }) => {
```

Comment:
use one line returning style

Reply:
is your arg for many days or just for one day?

Reply:
for many days